packages/js/block-templates/src/hooks/use-woo-block-props/test/use-woo-block-props.test.ts

```diff
@@ -31,6 +31,7 @@ describe( 'useWooBlockProps', () => {
 		expect( useBlockProps ).toHaveBeenCalledWith( {
 			'data-template-block-id': 'test/block',
 			'data-template-block-order': 30,
+			tabIndex: -1,
```

Comment:
Is it going to set `tabindex="-1"` by default? Without know deeply the editor implementation, I'd say that maybe it should be handled only for these component instances?

Reply:
It's ok to add tabindex -1 by default, that's what we want, since focusing on the "root" of the block doesn't have any purpose in our context.